ci-operator/config/wainersm/kc-runtime/wainersm-kc-runtime-master.yaml

```diff
@@ -0,0 +1,63 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+  base-machine:
+    name: fedora
+    namespace: openshift
+    tag: "29"
+  machine-os-content-base:
+    name: "4.3"
+    namespace: ocp
+    tag: machine-os-content
+binary_build_commands: .ci/openshift-ci/build.sh
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .ci/openshift-ci
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/wainersm/kc-runtime/_out
+    machine-os-content-base:
+      as:
+      - registry.svc.ci.openshift.org/openshift/origin-v4.0:machine-os-content
+      paths: null
+  to: disabled-machine-os-content
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+tests:
+- artifact_dir: /go/src/github.com/kata-containers/runtime/_out/test-artifacts
+  as: e2e-periodic-azure-nested-virt
+  commands: |
+    env
+    .ci/openshift-ci/test.sh
+  openshift_installer_custom_test_image:
+    cluster_profile: azure4
+    from: pipeline:bin
+    nested_virt_image: rhcos43-nested-virt
```

Comment:
When you use the azure provider you won't need the `nested_virt_image` parameter any more.